src/models/util.ts

```diff
@@ -0,0 +1,35 @@
+import * as Knex from 'knex';
+import { intersection, compose } from 'ramda';
+import { CommonQueryOptions } from './models';
+
+
+export const Utils: {
+  [k: string]: (...a: any[]) => (q: Knex.QueryBuilder) => Knex.QueryBuilder
+} = {
+  limit: (n: number) => (q: Knex.QueryBuilder) => q.limit(n),
```

Comment:
What does this type signature mean?

Reply:
```
limit :: Number -> QueryBuilder -> QueryBuilder
```
It just returns a modified `QueryBuilder` instance.